lib/desired-caps.js

```diff
@@ -34,6 +34,9 @@ let espressoCapConstraints = {
   },
   intentOptions: {
     isObject: true
+  },
+  launchDisplayId: {
```

Comment:
what if we create an `activityOptions` object instead? My point is that ActivityOptions object in Android may contain several options, so there is no point in having a separate capability for each of these: https://android.googlesource.com/platform/frameworks/base/+/master/services/core/java/com/android/server/am/ActivityManagerShellCommand.java#487

Reply:
Yes, That is what i wanted.. but that is out of my current skills. Can anyone help on this?

Reply:
that's easy. Simply follow the way it's done for `intentOptions`

Reply:
Let me try to create my first "hello world" by kt.